Bug: Shiki cannot be used with --isolatedModules flag

This was introduced between 0.9.3 and 0.9.5. TypeScript is complaining about `FontStyle` here.

```
node_modules/shiki/dist/index.d.ts:269:68 - error TS2748: Cannot access ambient const enums when the '--isolatedModules' flag is provided.   

269 export { languages as BUNDLED_LANGUAGES, themes as BUNDLED_THEMES, FontStyle, Highlighter, HighlighterOptions, HtmlRendererOptions, ILanguageRegistration, IShikiTheme, IThemeRegistration, IThemedToken, Lang, Theme, getHighlighter, fetchTheme as loadTheme, renderToHtml, setCDN, 
setOnigasmWASM };
```

--- a/packages/shiki/src/stackElementMetadata.ts
+++ b/packages/shiki/src/stackElementMetadata.ts
@@ -40,7 +40,7 @@ export const enum TemporaryStandardTokenType {
   MetaEmbedded = 8
 }
 
-export const enum FontStyle {
+export enum FontStyle {
   NotSet = -1,
   None = 0,
   Italic = 1,

